src/view/configuration/configuration.jsx

```diff
@@ -39,10 +39,11 @@ const contextGranularityEnum = {
 };
 const contextOptions = ["web", "device", "environment", "placeContext"];
 
-const instanceDefaults = {
+const getInstanceDefaults = initInfo => ({
```

Comment:
Are the parens needed here? I may be wrong, but I think it would return {} normally.

Reply:
Without the parenthesis, the browser sees the curly braces as part of a regular function (e.g., `(initInfo) => { ... }` and will throw an error when it parses `name: "alloy"`. See the Advanced Syntax section here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions

Reply:
Oh, thanks. I didn't notice that